413_2bA100000.geojson

```diff
@@ -0,0 +1,1926 @@
+{
+  "type": "FeatureCollection",
+  "features": [
+  {
+      "type": "Feature",
+      "id": "am002707",
+      "geometry": {
+        "type": "MultiPolygon",
+        "coordinates": [
+    [
+        [
+            [
+                34.88861,
+                33.29528
+            ],
+            [
+                34.88861,
+                32.93417
+            ],
+            [
+                35.31667,
+                32.93417
+            ],
+            [
+                35.31667,
+                33.29528
+            ],
+            [
+                34.88861,
+                33.29528
+            ]
+        ]
+    ]
+]},
+        "geometry_name": "geom",
+        "properties": {
+    "label": "1",
+    "datePub": "1976",
+    "west": 34.88861,
+    "east": 35.31667,
+    "north": 33.29528,
+    "south": 32.93417,
+    "scale": "100000",
+    "color": "31",
+    "dateSurvey": "1975",
+    "publisher": "Survey of Israel.",
+    "available": false,
+    "digHold": "https://collections.lib.uwm.edu/digital/collection/agdm/id/25000/rec/1",

```

Comment:
These URLS do not need the `/rec/#` endings on them. Basically those get added on when you page through search results.

@PaulLee98 could you modify this file or edit the URLS in the table and re-run the script. Whichever you think will be easier. 

So for example this one is [https://collections.lib.uwm.edu/digital/collection/agdm/id/25000/rec/1](https://collections.lib.uwm.edu/digital/collection/agdm/id/25000/rec/1) which does work, but [https://collections.lib.uwm.edu/digital/collection/agdm/id/25000/](https://collections.lib.uwm.edu/digital/collection/agdm/id/25000/) works too.